plugins/notifications-backend/src/service/router.ts

```diff
@@ -177,9 +180,34 @@ export async function createRouter(
     return users;
   };
 
+  const filterProcessors = (payload: NotificationPayload) => {
+    const result: NotificationProcessor[] = [];
+
+    for (const processor of processors) {
+      if (processor.getNotificationFilters) {
+        const filters = processor.getNotificationFilters();
+        if (filters.minSeverity) {
+          const payloadSeverity = payload.severity
+            ? payload.severity
+            : ('normal' as NotificationSeverity);
+          if (
+            notificationSeverities.indexOf(payloadSeverity) >
```

Comment:
Why not just `payload.severity ?? 'normal'` like it's done in the send handler?

Reply:
fixed